samples/excel/85-preview-apis/dynamic-arrays.yaml

```diff
@@ -0,0 +1,185 @@
+            const spillRange = currentRange.getSpillingToRangeOrNullObject();
+            spillRange.load("address");
+            await context.sync();
+
+            // Log the dynamic array information contained within the selected cell.
+            if (!spillParent.isNullObject) {
+              console.log(`The selected range has a spill parent at ${spillParent.address}`);
+            }
+            if (!spillRange.isNullObject) {
+              console.log(`The selected range is spilling into ${spillRange.address}`);
+            }
+
+            if (spillParent.isNullObject && spillRange.isNullObject) {
+              console.log("The selected range is not involved with any dynamic arrays. Be sure only one cell is selected.");
+            }
+          });
+        }
+
+        /** Create a new table with sample data */
+        async function setup() {
+          await Excel.run(async (context) => {
+            context.workbook.worksheets.getItemOrNullObject("Sample").delete();
+            const sheet = context.workbook.worksheets.add("Sample");
+
+            const expensesTable = sheet.tables.add("A4:D4", true /*hasHeaders*/);
+            expensesTable.name = "ExpensesTable";
+
+            expensesTable.getHeaderRowRange().values = [["Date", "Merchant", "Category", "Amount"]];
+
+            expensesTable.rows.add(null /*add at the end*/, [
+              ["1/1/2020", "The Phone Company", "Communications", "$120"],
+              ["1/2/2020", "Northwind Electric Cars", "Transportation", "$142"],
+              ["1/5/2020", "Best For You Organics Company", "Groceries", "$27"],
+              ["1/10/2020", "Coho Vineyard", "Restaurant", "$33"],
+              ["1/11/2020", "Bellows College", "Education", "$350"],
+              ["1/15/2020", "Trey Research", "Other", "$135"],
+              ["1/15/2020", "Best For You Organics Company", "Groceries", "$97"]
+            ]);
+
+            sheet.getRange("A2:H2").values = [["Transactions", , , , , , "Category", "Groceries"]];
+            sheet.getRange("A2").style = "Heading1";
+            sheet.getRange("G2").style = "Heading2";
+            sheet.getRange("H2").format.fill.color = "#EEEE99";
+
+            sheet.getUsedRange().format.autofitColumns();
+            sheet.getUsedRange().format.autofitRows();
+
+            sheet.activate();
+            await context.sync();
+          });
+        }
+
+        /** Default helper for invoking an action and handling errors. */
+        async function tryCatch(callback) {
+          try {
+            await callback();
+          } catch (error) {
+            // Note: In a production add-in, you'd want to notify the user through your add-in's UI.
+            console.error(error);
+          }
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+          <p>This sample shows how to apply formulas that return dynamic arrays and show how to get the relevant information
```

Comment:
```suggestion
          <p>This sample shows how to apply formulas that return dynamic arrays and how to get the relevant information
```

Reply:
Intro sentence is unclear to me, is the second 'show' necessary? If so, what is doing the showing?

Reply:
Nope! Good catch.